docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs crontabber.
+
+# Run crontabber sleeping 5 minutes between runs
+while true; do
+    python socorro/cron/crontabber_app.py
+    echo "Sleep 5 minutes..."
+    sleep 300
+done
```

Comment:
It's late Friday but isn't it a good thing that it doesn't continue to run if the configuration is so broken that crontabber can't get up and running. All "runtime exceptions" when executing crontabber apps are all captured and dealt with. 

It's a bit akin to how you start celery. You don't want a celery working to start without saying anything if it can't connect to the service where it gets jobs from. But when it's up and running it doesn't stop running just because a task within raises an exception.

I admit, this is a drive-by comment. I haven't really booted my brain as to how this'll run/work in a local docker dev world.